src/component/Form.js

```diff
@@ -31,12 +32,16 @@ class Form extends React.Component {
         window.alert(response.result.error.message)
       } else {
         const sortedVideos = response.result.items.sort((a, b) => (parseInt(a.statistics.viewCount) < parseInt(b.statistics.viewCount)) ? 1 : -1)
+        this.setState({ showError: false })
         videoList.set(sortedVideos)
       }
-      isLoading.set(false)
     } catch (error) {
-      console.log(error)
+      console.log(error.body)
+      this.errorMessage = error.body
+      videoList.set([])
```

Comment:
why?

Reply:
why what

Reply:
or the videoList remain whatever the last time was